FreeRTOS/FreeRTOS
# Removed issue template for general inquiries

Description
-----------
Removing issue template for general inquiries. General inquiries are redirected to forum. 

(Note that, it seems I cannot leave the "general inquiries" entry and disable to button. So the easiest way is really just remove the template.)

Test Steps
-----------
When a user is trying to create a PR, only providing the options for creating bug report or documentation issue report. Here's what the landing page looks like:
![Screen Shot 2020-06-23 at 11 03 23 PM](https://user-images.githubusercontent.com/10982575/85506853-0584a880-b5a6-11ea-80aa-b51887bc995e.png)

With bug report and documentation issue report template, the last line is hidden. See:
https://github.com/yuhui-zheng/FreeRTOS/issues/1

Related Issue
-----------
N/A


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
